### DISCUSSION TITLE
Trained epoches of pretrained model stt_en_conformer_ctc_medium_ls

### DISCUSSION THREAD

**Discussion by yiyibooks** [NVIDIA-NeMo/NeMo#2513] [Q&A] [answered]
Hello,

I'm trying to train conformer_ctc_medium model from scratch.  I use the config file from the pretrained model stt_en_conformer_ctc_medium_ls and only change the manifest_filepath and batch_size  items (I changed the batch_size to 16 to fit my GPU).

After training 42 epoches, the WER on librispeech validate data is still much higher than the pretrained stt_en_conformer_ctc_medium_ls model. My trained models WER on  librispeech validate data dev-other is about 15%.

May I know the trained epoch of the pretrained model stt_en_conformer_ctc_medium_ls ? Or is there any other configuration item I should change to acheive similar results as the pretrained model ?

Thank you very much!

**Comment by VahidooX** [comment, marked as answer]
https://github.com/NVIDIA/NeMo/issues/2525
